apps/web/components/eventtype/CreateEventType.tsx

```diff
@@ -51,9 +51,7 @@ export default function CreateEventTypeButton(props: Props) {
 
   // URL encoded params
   const teamId: number | undefined =
```

Comment:
Why leave 53 with teamId if we're no longer using it? It's initialized as const as empty?